com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java

```diff
@@ -102,35 +108,24 @@ else if (!isExistingTemplate(template)) {
 		projectTemplatesArgs.setService(_options.service());
 		projectTemplatesArgs.setTemplate(template);
 
-		execute(projectTemplatesArgs, isWorkspace);
+		execute(projectTemplatesArgs);
 
 		_blade.out().println(
-			"Created the project " + projectTemplatesArgs.getName() + " using the " +
-					projectTemplatesArgs.getTemplate() + " template in " + dir);
+			"Created the project " + projectTemplatesArgs.getName() +
+			" using the " + projectTemplatesArgs.getTemplate() +
+			" template in " + dir.getAbsolutePath());
 	}
 
-	void execute(ProjectTemplatesArgs projectTemplatesArgs, boolean isWorkspace) throws Exception {
+	void execute(ProjectTemplatesArgs projectTemplatesArgs) throws Exception {
 		File dir = projectTemplatesArgs.getDestinationDir();
+		String name = projectTemplatesArgs.getName();
 
 		new ProjectTemplates(projectTemplatesArgs);
 
-		if (isWorkspace) {
```

Comment:
And now , I can make sure that new ProjectTemplates(ProjectTemplatesArgs)  will not extract wrapper.zip if it is in liferay-workspace see https://github.com/liferay/liferay-portal/blob/master/modules/sdk/project-templates/src/main/java/com/liferay/project/templates/ProjectTemplates.java#L177-L196

So I delete the deleting code in our blade code.